### ISSUE TITLE
Docker?

### ISSUE THREAD

**Issue by wangshun1121** [dmnfarrell/smallrnaseq#9]
Should there be a docker image of this tool?

**Comment by dmnfarrell** [comment]
Yes there probably should but I haven't thought of making one. If you know how please let me know!

**Comment by wangshun1121** [comment]
OK, I'll fork this repo to mine, and try to develop a docker image. I'll let you know if I complete a fine image!

**Comment by wangshun1121** [comment]
@dmnfarrell 

I developed a docker image, using following CMD to pull and have a try:

```
docker pull wangshun1121/smallrnaseq:latest 
```

The docker file is https://github.com/wangshun1121/smallrnaseq/blob/master/Dockerfile

**Comment by dmnfarrell** [comment]
How do I run this?

**Comment by wangshun1121** [comment]
Try following CMDs:

```
docker run -it wangshun1121/smallrnaseq:latest  bash
```

then you'll enter the container of small RNA analysis environment, and see whether your tools work.

By the way, I develop this image using python3 environment, and in your scripts, there are few codes using python2. I've modified your codes in my repo, you can check up these codes in git commit info.
